From 3ea54def194ee7c79b3ade000825fdad07603d24 Mon Sep 17 00:00:00 2001
From: Michel Thebeau <Michel.Thebeau@windriver.com>
Date: Fri, 29 Sep 2023 21:23:19 +0000
Subject: [PATCH] Add log level option for vault-manager

Add manager.log.defaultLogLevel to values.yaml to allow a developer to
set the log level of vault-manager pod.  Values may be:

  DEBUG: 1
  INFO: 2   (default)
  WARNING: 3
  ERROR: 4
  FATAL: 5

Signed-off-by: Michel Thebeau <Michel.Thebeau@windriver.com>
---
 values.yaml | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/values.yaml b/values.yaml
index f35df52..600d632 100644
--- a/values.yaml
+++ b/values.yaml
@@ -114,6 +114,15 @@ manager:
     #      client_version: v1.28
     client_version: ""
 
+  # Debugging option to improve log reading, allow more verbose logging
+  #   DEBUG: 1
+  #    INFO: 2
+  # WARNING: 3
+  #   ERROR: 4
+  #   FATAL: 5
+  log:
+    defaultLogLevel: 2
+
 injector:
   # True if you want to enable vault agent injection.
   # @default: global.enabled
-- 
2.34.1

